src/applications/caregivers/tests/e2e/1010cg.cypress.spec.js

```diff
@@ -0,0 +1,164 @@
+import path from 'path';
+
+import testForm from 'platform/testing/e2e/cypress/support/form-tester';
+import { createTestConfig } from 'platform/testing/e2e/cypress/support/form-tester/utilities';
+
+import formConfig from '../../config/form';
+import manifest from '../../manifest.json';
+
+const veteranLabel = `Enter Veteran\u2019s or service member\u2019s full name`;
+const primaryLabel = 'Enter Primary Family Caregiver\u2019s full name';
+const secondaryOneLabel = 'Enter Secondary Family Caregiver\u2019s full name';
+const secondaryTwoLabel =
+  'Enter Secondary Family Caregiver\u2019s (2) full name';
+
+const testSecondaryTwo = createTestConfig(
+  {
+    dataPrefix: 'data',
+    dataSets: [
+      'requiredOnly.json',
+      'oneSecondaryCaregivers.json',
+      'twoSecondaryCaregivers.json',
+    ],
+    fixtures: {
+      data: path.join(__dirname, 'fixtures', 'data'),
+      mocks: path.join(__dirname, 'fixtures', 'mocks'),
+    },
+
+    setupPerTest: () => {
+      cy.route('GET', '/v0/feature_toggles?*', 'fx:mocks/feature-toggles');
+    },
+    pageHooks: {
+      introduction: () => {
+        // Hit the start button
+        cy.findAllByText(/start/i, { selector: 'button' })
+          .first()
+          .click();
+      },
+      'review-and-submit': () => {
+        cy.get('@testKey').then(testKey => {
+          switch (testKey) {
+            case 'oneSecondaryCaregivers.json':
+              // sign signature as veteran
+              cy.get(`[data-test-id="${veteranLabel}-signature-input"]`)
```

Comment:
Will [`findByTestId`](https://testing-library.com/docs/dom-testing-library/api-queries#bytestid) work for these?

More importantly, the queries in the DOM testing library are [listed in order of recommendation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/testing/end-to-end/cypress-migration-guide.md#interacting-with-page-elements-). So if there's a label that goes with this input, it's usually better to match by that.

Reply:
I meant to use this, refactored and code is much cleaner and more readable, great catch thank you @U-DON!